test/Mono.Linker.Tests.Cases.Expectations/Assertions/LogContainsExactAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Assertions
+{
+	[AttributeUsage (AttributeTargets.Class, AllowMultiple = true, Inherited = false)]
+	public class LogContainsExactAttribute : EnableLoggerAttribute
```

Comment:
Maybe we should just add a parameter to the existing attribute:
* `LogContains("Text")` - would do `string.Contains`
* `LogContains("Test", RegularExpression = true) - would do the existing regex match

I think that is cleaner than having another attribute